jsonnet/jsonnetfile.lock.json

```diff
@@ -34,12 +34,12 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/coreos/prometheus-operator.git",
+          "remote": "https://github.com/coreos/prometheus-operator",
```

Comment:
shouldn't it be?

```suggestion
          "remote": "https://github.com/prometheus-operator/prometheus-operator",
```

Reply:
indeed, great catch, I focused on the version bump only :+1:

Reply:
fixed (it was a long way)